src/analysis/retail/mage/frost/apl/SpellslingerAplCheck.tsx

```diff
@@ -0,0 +1,32 @@
+import aplCheck, { build } from 'parser/shared/metrics/apl';
+import TALENTS from 'common/TALENTS/mage';
+import * as cnd from 'parser/shared/metrics/apl/conditions';
+import SPELLS from 'common/SPELLS';
+
+export const spellslingerApl = build([
+  {
+    spell: TALENTS.FLURRY_TALENT,
+    condition: cnd.and(
+      cnd.debuffMissing(SPELLS.WINTERS_CHILL),
+      cnd.or(cnd.lastSpellCast(SPELLS.FROSTBOLT), cnd.lastSpellCast(TALENTS.GLACIAL_SPIKE_TALENT)),
+    ),
+  },
+  {
+    spell: TALENTS.ICE_LANCE_TALENT,
+    condition: cnd.or(
+      cnd.debuffPresent(SPELLS.WINTERS_CHILL),
+      cnd.buffPresent(TALENTS.FINGERS_OF_FROST_TALENT),
+    ),
+  },
+  {
+    spell: TALENTS.FLURRY_TALENT,
+    condition: cnd.buffPresent(TALENTS.ICY_VEINS_TALENT),
+  },
+  {
+    spell: TALENTS.GLACIAL_SPIKE_TALENT,
+    condition: cnd.buffStacks(SPELLS.ICICLES_BUFF, { atLeast: 5 }),
+  },
```

Comment:
Flurry in IV condition should be behind GS, not in front of it

Reply:
thanks for pointing it out